docs/operate/get-started/other-hardware/_index.md

```diff
@@ -65,9 +65,9 @@ The lifecycle of a module and the resources it provides is as follows:
 1. `viam-server` starts, and if it is connected to the internet, it checks for configuration updates.
 1. `viam-server` starts any configured modules.
 1. When a module initializes, it registers its model or models and associated [APIs](/dev/reference/apis/) with `viam-server`, making the models available for use.
-1. For each modular resource configured on the machine, `viam-server` uses the resource's `validate` function and the `depends_on` field in the resource configuration to determine the dependencies of the resource.
+1. For each modular resource configured on the machine, `viam-server` uses the resource's `validate` function and the `depends_on` field in the resource configuration to determine the [dependencies](/operate/get-started/other-hardware/dependencies/) of the resource.
 1. If a dependency is not already running, `viam-server` starts it before starting the resource.
```

Comment:
```suggestion
1. If a required dependency is not already running, `viam-server` starts it before starting the resource.
```
This line is also only true for required dependencies. _Optional_ dependencies do not affect build order.

Reply:
Oh, wait so, even if an optional dependency *is* immediately available, viam-server won't build it first?

Reply:
Ah okay I think I'll add an explicit sentence about this to the dependencies page. Thanks!